packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -39,7 +40,7 @@ export function getQualityPercentagesRounded(invalid, empty, valid, digits = 0)
 	return [invalidRounded, emptyRounded, validRounded];
 }
 
-export function QualityBar({ invalid, valid, empty, digits = 1 }) {
+export function QualityBar({ invalid, valid, empty, onClick, dataFeature, digits = 1 }) {
```

Comment:
As i read the dataFeature attribute, i was surprised it was a function.
On other component, dataFeature props is usually a string and be composed in the component.

1) Having dataFeature attribute with a string composition with :
dataFeature={dataFeature ? `${dataFeature}-invalid` : null}

2) Having a bit of renaming like `getDataFeature` could be more explicit that this is a function

Reply:
i am agree, it's more isefull like this